fe/fe-core/src/main/java/com/starrocks/catalog/mv/MVTimelinessArbiter.java

```diff
@@ -204,5 +204,20 @@ protected void addEmptyPartitionsToRefresh(MvUpdateInfo mvUpdateInfo) {
                 mvUpdateInfo.addMvToRefreshPartitionNames(mvPartitionName);
             }
         });
+        Map<Table, List<Column>> refBaseTableAndColumns = mv.getRefBaseTablePartitionColumns();
```

Comment:
This compensation should be added both for `external` or `olap` tables? can we add another function to support this.

Reply:
done